TeadsSDKDemo/buildSrc/build.gradle.kts

```diff
@@ -1,13 +1,16 @@
 plugins {
     `kotlin-dsl`
+    id("org.jetbrains.kotlin.jvm") version "1.4.32"
 }
 
+dependencies {
+    implementation("com.android.tools.build:gradle:3.5.4")
```

Comment:
The error is related to the gradle android plugin version. Versions equal or greater than 7.0.0 handle correctly libraries that share same module name and build type release name

`implementation("com.android.tools.build:gradle:7.0.0")`

Could you try it?